ui/app/components/clients/total-client-usage.js

```diff
@@ -72,7 +72,7 @@ export default class TotalClientUsage extends Component {
     chartSvg.attr('viewBox', `0 5 725 305`); // set aspect ratio
 
     let groups = chartSvg
-      .selectAll('g')
+      .selectAll('g.rect')
```

Comment:
we should probably remove this - I don't think it made a difference and might mess with stuff later